include/z64effect.h

```diff
@@ -227,11 +227,11 @@ typedef struct {
 } EffectInfo; // size = 0x14
 
 typedef enum {
-    /* 0x00 */ EFFECT_SPARK,
-    /* 0x01 */ EFFECT_BLURE1,
-    /* 0x02 */ EFFECT_BLURE2,
-    /* 0x03 */ EFFECT_SHIELD_PARTICLE,
-    /* 0x04 */ EFFECT_TIRE_MARK
+    /* 0x0 */ EFFECT_SPARK,
+    /* 0x1 */ EFFECT_BLURE1,
+    /* 0x2 */ EFFECT_BLURE2,
+    /* 0x3 */ EFFECT_SHIELD_PARTICLE,
+    /* 0x4 */ EFFECT_TIRE_MARK
 } EffectType;
```

Comment:
```suggestion
    /* 0 */ EFFECT_SPARK,
    /* 1 */ EFFECT_BLURE1,
    /* 2 */ EFFECT_BLURE2,
    /* 3 */ EFFECT_SHIELD_PARTICLE,
    /* 4 */ EFFECT_TIRE_MARK
} EffectType;
```
I think single-digit enums don't need hex

Reply:
Enums I'm less concerned about, actually. I'd like to be consistent, but I don't mind if we don't use 0x for the small ones.

Reply:
Oh, I misread/misunderstood something, my bad. I'll adjust the enum comments.